classroom

```diff
@@ -0,0 +1,14 @@
+class Classroom
+	attr_accessor :label
+	attr_reader :students
+
+	def initialize(label)
+		@label = label
+		@student = []
+	end
+
+	def add_student(student)
+		@student.push(student)
+		@student.classroom = self
+	end
+end
```

Comment:
- [ ] Everything is looking good, nice job! However, the `.rb` extension for this file is missing, please add it and you will be done. 😎